src/assets/i18n/fr_FR.json

```diff
@@ -297,6 +298,20 @@
   "account.organization.cost_center_management.create.confirmation": "Le centre de coûts \"{{0}}\" a été créé.",
   "account.organization.cost_center_management.delete.confirmation": "Le centre de coûts a été supprimé.",
   "account.organization.cost_center_management.update.confirmation": "Le centre de coûts \"{{0}}\" a été mise à jour.",
+  "account.organization.hierarchies": "Structure de la société",
+  "account.organization.hierarchies.grouppath.tooltip.close": "Réduire",
+  "account.organization.hierarchies.grouppath.tooltip.open": "Afficher tous les groupes",
+  "account.organization.hierarchies.groups.button.add.label": "Ajouter un groupe",
+  "account.organization.hierarchies.groups.delete.confirmation": "Le groupe dont l’identifiant est \"{{0}}\" a été supprimé.",
+  "account.organization.hierarchies.groups.new.button.label": "Créer",
+  "account.organization.hierarchies.groups.new.confirmation": "Le groupe \"{{0}}\" a été créé.",
+  "account.organization.hierarchies.groups.new.message": "Création d’un nouveau groupe reflétant la structure de votre entreprise. Vous pouvez y attribuer des utilisateurs.",
```

Comment:
see comment for EN

Reply:
Create a new group to reflect your company structure.
Créez un nouveau groupe pour refléter la structure de votre entreprise.

> correct?

Reply:
Arvid confirmed the following: 
Créez un nouveau groupe pour refléter la structure de votre societé.